README.md

```diff
@@ -30,24 +29,48 @@
 ### npm
 
 ```sh
-> npm install rif-communications
+> npm install @rsksmart/rif-communications
 ```
 
-### Use in Node.js
-
 ## Usage
 
+```js
+// 1. Import the library
+var RifCommunications = require('@rsksmart/rif-communications');
+
+// 2. Create a key or load from a secure storage to skip this step
+var key = await RifCommunications.createKey();
+
+// 3. Create peer id
+var peerId = await RifCommunications.createPeerIdFromJSON(key);
+
+// 4. Create peer information
+var peerInfo = await RifCommunications.createPeerInfo(peerId);
+
+// 5. Create your peer node
+var node = RifCommunications.createNode(peerInfo, host, port, sendMsgFunc);
+
+// 6. Connect to a bootnode using a multiaddress string
+await RifCommunications.connectToNode(node, BOOTNODE_ADDRESS);
+
+// 7. From the public key of the recipient create the recipient
+var recipient = await RifCommunications.createPeerIdFromPublicKey(recipientID);
+
+// 8. Send a message
+await RifCommunications.sendMsg(node, recipient, message, msgNonce);
+```
+
+
 ## API
 
 ## Contribute
 
 There are some ways you can make this module better:
 
-- Consult our [open issues](https://github.com/rsksmart/rif-communications-js/issues) and take on one of them
+- Consult our [open issues](https://github.com/rsksmart/rif-communications/issues) and take on one of them
 - Help our tests reach 100% coverage!
```

Comment:
> How you can improve this module:
> - Consult our [open issues](https://github.com/rsksmart/rif-communications/issues) page and pick one
> - Help us reach 100% test coverage!

Reply:
Isn't it still `rif-communications-js/issues`? Why was this line changed?

Reply:
I think same @jurajpiar but that is a part of our Readme template, we can discuss about this. @vojtechsimetka: I don't know how it happened